starlette/testclient.py

```diff
@@ -243,6 +245,7 @@ def handle_request(self, request: httpx.Request) -> httpx.Response:
                 "client": ["testclient", 50000],
                 "server": [host, port],
                 "subprotocols": subprotocols,
+                "state": self.state,
```

Comment:
I'm not seeing where this dict is copied. Is it being copied before being passed into each request?

Reply:
Ups :eyes: